docs/geedocs/answer/4492599.html

```diff
@@ -90,7 +93,7 @@
 <p><strong>Browsers</strong></p>
 <p>Google Earth Enterprise is supported on the following browsers on all operating systems:</p>
 <ul>
-<li dir="ltr">Google Chrome version 10 and later (32-bit)</li>
+<li dir="ltr">Google Chrome version 10 and later</li>
```

Comment:
Do we have document somewhere (perhaps on the Git Hub Wiki) mentioning what OS distributions, and browser versions we are supporting? Otherwise, I'm not sure how accurate this document is going to be.